client/components/editor/editor-code.vue

```diff
@@ -233,6 +233,9 @@ export default {
           if (opts.align && opts.align !== '') {
             img += ` class="align-${opts.align}"`
           }
+          if (opts.className && opts.className !== '') {
+            img += ` class="align-${opts.className}"`
+          }
```

Comment:
This would add a duplicate class if align is also set.

Reply:
![image](https://user-images.githubusercontent.com/7647943/166874518-9559bb81-ff50-4068-9c00-beca96ffce21.png)

It will render something like this if both options are added

Reply:
I'm not sure, why we can't use `:class="{'classname': condition, ... }"` syntax here, but its probably a pug thing.
Maybe using function from [classNames](https://github.com/JedWatson/classnames) module might be a solution here. 

Anyways the logic is not too simple and might be moved to it's own function.

`
function getClassProp(opts) {
    if (!opts || (!opts.align && !opts.className)) {
        return '';
    }

    const classes = [];

    if (opts.align) {
        classes.push(`align-${opts.align}`);
    }

    if (opts.className) {
        classes.push(opts.className);
    }

    const classProp = classes.length > 0 ? ` class="${classes.join(' ')}"` : '';

    return classProp;
}
`